plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ScriptMediatorPropertiesEditionPartForm.java

```diff
@@ -136,6 +136,8 @@
     protected GridData scriptKeysData;
     
     protected Composite propertiesGroup;
+    
+    protected Composite filterScriptTypeSubPropertiesGroup;
```

Comment:
Some of these code bits are auto-generated, that's why they are formatted in a different way. So some of the formatting corrections were skipped **intentionally** because they will be overwritten every time the code is generated.